### DISCUSSION TITLE
Best way to organize images in gallery / tag

### DISCUSSION THREAD

**Discussion by InterstellarOne** [deckerst/aves#1334] [Q&A]
I have a series of images that I would like to display in a specific order in my gallery, but since I migrated them from a different gallery they imported out of order. What's the best way to get them into order? Each image is numbered, but the file names are random hashes.

**Comment by deckerst** [comment]
> Each image is numbered

Numbered with the random hash, right? Or the number is in some kind of metadata field? 

And I understand that the images didn't have a metadata date. So you're left with files with no name and no meaningful date. In that case, I only see manual steps to give it order (either individual renaming or individual date editing). I don't think the bulk renaming feature would help here.
